roles/calico/tasks/main.yml

```diff
@@ -5,7 +5,9 @@
   when: (calico_etcd_cert_dir is defined or calico_etcd_ca_cert_file is defined or calico_etcd_cert_file is defined or calico_etcd_key_file is defined or calico_etcd_endpoints is defined) and not (calico_etcd_cert_dir is defined and calico_etcd_ca_cert_file is defined and calico_etcd_cert_file is defined and calico_etcd_key_file is defined and calico_etcd_endpoints is defined)
 
 - name: Calico Node | Generate OpenShift-etcd certs
-  include: ../../../roles/etcd_client_certificates/tasks/main.yml
+  include_role:
+    name: etcd
+    tasks_from: client_certificates
```

Comment:
I like this format.

Reply:
Let's move this include_role into the play that calls calico role.